app/models/transaction.rb

```diff
@@ -79,6 +89,7 @@ def update_balance_in_account
     #
     Transaction.transaction do
       account.save
+      raise ActiveRecord::Rollback
```

Comment:
@babarajay , Wouldn't it always raise an exception?

Reply:
if we raise an ActiveRecord::Rollback exception, then the database transaction will be rolled back, without passing on the exception.

Reply:
@babarajay , Yes. it's okay. But won't it always raises an exception? There isn't any condition

Reply:
I think; I'm not getting what exactly you're trying to point out here. While saving the money transfer if there any issue occurs; it will rollback the transaction to previous state of the object. Isn't it?

Reply:
Kind request to resolve my query.